### ISSUE TITLE
Issue with the stroke around the buttons

### ISSUE THREAD

**Issue by akathewb** [joezcool02/OurCalculator#69]
When I first start the calculator all the stroke lines are around the buttons, then after a bit of time passes making calculations the stroke of the shapes end up all  pulled to one side. ( see image below)

![image](https://cloud.githubusercontent.com/assets/23046037/26708161/f3a2ba08-46fe-11e7-8c8a-1e3267c9b065.png)

**Comment by joseph-allen** [comment]
incredible! how do you trigger this just perform a few sums?

**Comment by ayush-seth** [comment]
Can you consistently and without variation, reproduce this bug every time you run the calculator?

**Comment by joseph-allen** [comment]
it's common on projects like this to comment here if you can reproduce it,
then thumb that comment up if somebody else can reproduce and so on

**Comment by ayush-seth** [comment]
My calculator has been running for 20 minutes now doing calculations, not been able to reproduce this

**Comment by akathewb** [comment]
Yes, I can run the program and in less then 30 seconds (without interacting with the calculator) the stokes end up like my screen shot above. It seems that performing calculations does not seem to matter. 

So weird. I've tried to hunt for references to anything like this in other forums, but can find any. Was thinking of posting a processing github issue. I'm thinking it is not the calculator program but a rendering thing or graphics card thing.

**Comment by joseph-allen** [comment]
is it instantaneous or gradual?

wht resolution is your monitor?

**Comment by akathewb** [comment]
Instant.

I have two monitors one is the laptop monitor which is 1366x768, and the other is an LED Monitor 1600x900.

I'm running Windows 10, 4GB RAM, Intel(R) Celeron(R) CPU N2815 @ 1.86Ghz, 64-bit Operating System.

**Comment by joseph-allen** [comment]
our button class takes a fixed size, I assume at some point something is changing. we use rect() so we aren't even controlling the drawing of these lines ourself.

needs proper investigation really

**Comment by akathewb** [comment]
Also posted an issue here for reference:
https://github.com/processing/processing/issues/5109

What build of processing are you all using? (folks in the other forum asked if we are all using the same)
I'm running 3.3.3-64bit for Windows

**Comment by ayush-seth** [comment]
I am running the latest one

**Comment by Jitender46559** [comment]
In which language i have to write code can anyone answer
